assets/src/edit-story/components/devTools/devTools.js

```diff
@@ -114,6 +115,7 @@ function DevTools() {
     current,
     selection,
     story: { stylePresets },
+    version,
```

Comment:
We want the version as part of the debug screen so when we save off the story data we know which version it's in for migration.

Reply:
Can't we just add the version number to the debug output, rather than pollute the storage with it? It'll be a static number anyway for the duration of the visit. We can just do `import { DATA_VERSION } from '../../../migration';` in this file and output the number, right?